src/IdentityServer/Validation/Models/ValidatedBackchannelAuthenticationRequest.cs

```diff
@@ -83,4 +83,10 @@ public class ValidatedBackchannelAuthenticationRequest : ValidatedRequest
     /// Gets or sets the request object (either passed by value or retrieved by reference)
     /// </summary>
     public string? RequestObject { get; set; }
+
+    /// <summary>
+    /// Gets or sets a dictionary of custom properites that can store
+    /// additional state during the back channel authentication process.
+    /// </summary>
+    public Dictionary<string, object> Context { get; set; } = new();
```

Comment:
I feel `Context` isn't a good name for this. `Properties` would maybe be better.